common/djangoapps/course_modes/models.py

```diff
@@ -375,6 +402,132 @@ def min_course_price_for_currency(cls, course_id, currency):
+            enrollment_title = _("You're enrolled as an honor code student")
+            enrollment_value = _("Honor Code")
+        elif mode == "audit":
+            enrollment_title = _("You're auditing this course")
+            enrollment_value = _("Auditing")
+        elif mode in ["professional", "no-id-professional"]:
+            enrollment_title = _("You're enrolled as a professional education student")
+            enrollment_value = _("Professional Ed")
+        else:
+            enrollment_title = ''
+            enrollment_value = ''
+
+        return {
+            'enrollment_title': unicode(enrollment_title),
+            'enrollment_value': unicode(enrollment_value),
+            'show_image': show_image,
+            'image_alt': unicode(image_alt),
+            'display_mode': cls._enrollment_mode_display(mode, verification_status)
+        }
+
+    @staticmethod
+    def _enrollment_mode_display(enrollment_mode, verification_status):
+        """
+         return the of mode after checking enrollment mode and status.
+         Args:
+            enrollment_mode (str): enrollment mode.
+            verification_status (str) : verification status of student
+
+        Returns:
+            display_mode (str) : display mode for certs
+        """
+
+        # import inside the function to avoid the circular import
+        from student.helpers import (
+            VERIFY_STATUS_NEED_TO_VERIFY,
+            VERIFY_STATUS_SUBMITTED,
+            VERIFY_STATUS_APPROVED
+        )
+
+        if enrollment_mode == "verified":
+            if verification_status in [VERIFY_STATUS_NEED_TO_VERIFY, VERIFY_STATUS_SUBMITTED, VERIFY_STATUS_APPROVED]:
+                display_mode = "verified"
+            else:
+                display_mode = "honor"
+        elif enrollment_mode in ["professional", "no-id-professional"]:
+            display_mode = "professional"
+        else:
+            display_mode = enrollment_mode
+
+        return display_mode
+
+    @classmethod
+    def has_professional_mode(cls, modes_dict):
+        """
+        check the course mode is profession or no-id-professional
+
+        Args:
+            modes_dict (dict): course modes.
+
+        Returns:
+            bool
+        """
+        return "professional" in modes_dict or cls.NO_ID_PROFESSIONAL_MODE in modes_dict
+
+    @classmethod
+    def is_professional_mode(cls, course_mode_tuple):
+        """
+        checking that tuple is professional mode.
+        Args:
+            course_mode_tuple (tuple) : course mode tuple
+
+        Returns:
+            bool
+        """
+        return course_mode_tuple.slug in ['professional', cls.NO_ID_PROFESSIONAL_MODE] if course_mode_tuple else False
+
+    @classmethod
+    def is_professional_slug(cls, slug):
+        """
+        Args:
+            slug (str) : course mode string
+        Return:
+            bool
+        """
+        return slug in ['professional', cls.NO_ID_PROFESSIONAL_MODE]
```

Comment:
Also, it seems like it would make sense to group these helper functions with [those used to perform checks for verified modes](https://github.com/edx/edx-platform/pull/7267/files#diff-143f61740e18e9d6fe7ac104e7bee9a8R286).